markdownlint.js

```diff
@@ -182,9 +212,11 @@ program
   .version(pkg.version)
   .description(pkg.description)
   .usage('[options] <files|directories|globs>')
```

Comment:
The relevant part of the docs should be updated: https://github.com/igorshubovych/markdownlint-cli#usage

Reply:
Added the new option, updated the required/optional brackets, and added a short section to the README explaining the JUnit output.